### PR TITLE
use new mapps CLI for tunneling

### PR DESCRIPTION
**Pull request by gregra81**

### CODE REVIEW

**Comment by yuhgto** [comment]
For each sample app I did the following: 
- Removed the mtunnel package and any related code
- Added apps-cli package
- Run `mapps tunnel:create` in start script (or dev script, depending on how the project was structured)
- Update dependencies to remove vulnerabilities

File: apps/github-monday-code/SETUP.md [hunk 1]
```diff
@@ -24,8 +24,6 @@ To integrate with Github, you need to create an OAuth app using your Github acco
 
 1. Copy the url outputted by running the `npm run dev` command. For example: `https://happy-pig-99.tunnel.monday.app`.
```
Comment (gregra81) [inline apps/github-monday-code/SETUP.md:25]: @yuhgto also we should change the example here

**Review by gregra81 — COMMENTED** [review]
Amazing 🤩

File: apps/github-monday-code/SETUP.md (hunk 1 above)
Comment (yuhgto) [reply apps/github-monday-code/SETUP.md:25]: done

**Review by yuhgto — APPROVED** [review]
approved

**Pull request merged**